### DISCUSSION TITLE
Mail de confirmation quand ajouté.e à l'équipe de gestion ?

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1493] [Idées]
Je vois que notre équipe envoie des mails aux utilisateurs pour confirmer qu'iels ont été ajouté.e.s. Est-ce qu'on veut faire ça en manier automatique ?

**Comment by alemangui** [comment]
Aujourd'hui on envoie automatiquement l'email quand l'ajout est fait depuis l'app Vue (par un gestionnaire de la cantine). En ajoutant un trigger additionnel j'ai crains que certaines actions de masse déclenchent trop d'emails, mais à vérifier avec les bizdevs.

**Reply by gauthierandre** [reply]
Ça peut être top u'un mail de confirmation soit envoyé directement aux gestionnaires, sans qu'on soit en copie de ce courriel ? Ça éviterait effectivement de faire le courriel manuellement quand on s'occupe de sujet. Si j'ai bien compris, c'est déjà le cas quand un gestionnaire accepte l'ajout d'un nouvel utilisateur ?
